skbeam/core/tests/test_correlation.py

```diff
@@ -279,13 +279,13 @@ def test_CrossCorrelator1d():
                                        np.nan, 2.275846e-14]))
 
     assert_array_almost_equal(ycorr_1D_symavg[::20],
-                              np.array([3.347542,  1.54268227,  0.86220476,
+                              np.array([-5.3002753,  1.54268227,  0.86220476,
```

Comment:
I'm not following. I assume this test passed at some point in the past, so I'm wondering why one element would be off by 8.4.